### ISSUE TITLE
[A11y][Password Input] Label is truncated when zoom is applied

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#600]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S25

### OS Version

16

### App version

1.0.0

### Describe the issue

The password input label should be displayed entirely with text zoom at 200% (when input is not selected status)
<img width="317" height="403" alt="" src="https://github.com/user-attachments/assets/eff58203-2c22-48ef-814f-583d34154df9" />

### Expected Behavior

See example on native apps

<img width="331" height="158" alt="" src="https://github.com/user-attachments/assets/59393447-90cc-4797-b87b-d3f9792c00c5" />

**Comment by nouha06** [comment]
@pya35 according to [figma](https://www.figma.com/design/QtOWrH1m3RHOAkfyy0XFil/-OUDS-Lib--Components-library?node-id=95015-48288&m=dev) when zoom should apply ellipsis , so i think i should fixed like it mentioned on figma design

<img width="378" height="626" alt="Image" src="https://github.com/user-attachments/assets/067a83e6-8285-4d5f-9b2a-9ebd0194a426" />

**Comment by pya35** [comment]
@nouha06 @MaximeTonnerre  I'm not agree with Figma design about the ellipsis. With 200% zoom, all informative text shall be displayed entirely (label and helper text at least).

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (695)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 600
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)
